examples/grid_example/public/app.tsx

```diff
@@ -11,7 +11,7 @@ import deepEqual from 'fast-deep-equal';
 import { cloneDeep } from 'lodash';
 import React, { useCallback, useEffect, useMemo, useRef, useState } from 'react';
```

Comment:
starting a keyboard navigation event doesn't work properly for the main, but not first section (not collapsible)  (doesn't start properly) - we cannot move the element and the next section is selected:

Reply:
Fixed with 1bbbbde34845cc974b43950ef3164c8a3c5ad5d8 - thanks for catching this one, it was very sneaky 😆

As an added benefit, we can now use keyboard controls to change sections!! Remove the early return in 1ad5a31e94b2c70d2613fc946abea5df40e9ee43 🎉🎉 🎉